torchtext/experimental/datasets/language_modeling.py

```diff
@@ -229,6 +229,8 @@ def WMTNewsCrawl(tokenizer=None, root='.data', vocab=None, data_select=('train')
         >>> tokenizer = get_tokenizer("spacy")
         >>> train_dataset, = WMTNewsCrawl(tokenizer=tokenizer, data_select='train')
 
+    Note: unlike other language modeling datasets, WMTNewsCrawl provides a single dataset based on the year and language. There
```

Comment:
nit: WMTNewsCrawl provides datasets based on year and language instead of train/valid/test.